test/GoTrueClient.test.ts

```diff
@@ -280,6 +281,19 @@ describe('GoTrueClient', () => {
     expect(user?.email).toBe(email)
   })
 
+  test('signIn with OpenIDConnect', async () => {
+    const openIDConnect:OpenIDConnectCredentials = {
```

Comment:
```suggestion
    const oidc: OpenIDConnectCredentials = {
```

The use `oidc` in sign in.